transformations/negate_strengthen/transformation.py

```diff
@@ -0,0 +1,462 @@
+import random
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+import re
+import spacy
+from spacy.tokenizer import Tokenizer
+import pandas as pd
+import nltk
+from nltk.corpus import wordnet
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.metrics import edit_distance
+from nltk import pos_tag
+from pattern.en import conjugate as conjugate_en
+from typing import Union
+
+# defaults
+_RE_COMBINE_WHITESPACE = re.compile(r"\s+")
+MODAL_TYPE_LEMMA = ['be', 'to', 'have', 'get', 'do']
+MODAL_STRENGTHEN_DICT = {
+    'could': 'would',
+    'should': 'would',
+    'would': 'would',
+    'can': 'will',
+    'may': 'will',
+    'might': 'will',
+    'will': 'will'
+}
+
+# intialise
+try:
+    conjugate_en(verb='testing',tense='present',number='singular')
+except:
+    pass
+
+
+class NegateStrengthen(SentenceAndTargetOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.SENTIMENT_ANALYSIS]
+    languages = ["en"]
+    tgt_languages = ["en"]
+
+    def __init__(self, seed=0, verbose=False):
```

Comment:
Better to take 'max_outputs' as one of the params (which decides the max number of output, the generate function will return).

Reply:
Ok, done, please check implementation is what you mean